scripts/prepare-ci-environment.sh

```diff
@@ -10,8 +10,8 @@ fi
 # Python plugin for Community and Ultimate Edition
 
 export CWD=$(pwd)
-export IJ_VERSION="191.6183.62"
-export IJ_BUILD_NUMBER="191.6183.62"
+export IJ_VERSION="192.5728.98"
```

Comment:
```suggestion
export IJ_VERSION="2019.2"
```

Reply:
I've applied this suggestion and added a comment. This was causing `scripts/setup-ci-environment.sh` to fail with exit code 8 when attempting to download IntelliJ.